config/initializers/cypress_rails.rb

```diff
@@ -20,6 +20,10 @@
   else
     Rake::Task["db:seed:e2e"].invoke
   end
+
+  # [@jeremyf] <2022-02-07 Mon> :: With https://github.com/forem/forem/pull/16423 we assume that the
+  # listing feature must be explicitly enabled.
+  FeatureFlag.enable(:listing_feature)
```

Comment:
Do we need to put this into the initializer? Can we not just toggle the feature flag for the specs that need it? 

https://github.com/forem/forem/pull/16379

Reply:
We can absolutely toggle it for the two files that need it.